src/modules/actions.js

```diff
@@ -0,0 +1,180 @@
+        likesCount[i].textContent = likes;
+
+        await fetch(`${likesURL}${appID}/likes/`, {
+          method: 'POST',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify({ item_id: e.currentTarget.dataset.id }),
+        });
+      });
+    }
+    displayDetails();
+  });
+  const pageItems = document.querySelector('.home-count');
+  pageItems.textContent = `${counter(truncated)}`;
+};
+
+const displayDetails = () => {
+  const commentButtons = document.querySelectorAll('.comments-btn');
+  commentButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const projectDetails = document.querySelector('.popup-detail');
+      const main = document.querySelector('.major');
+      projectDetails.classList.add('showpopup');
+      main.classList.add('blur');
+      // console.log(button.id);
+      const movie = truncated[button.id - 1];
+      const popupDetail = `<article class="detail-container" id="popup">
+        <button type="button" id="popup-close" data-close-button class="close-button-project" >&times;</button>
+        <div class="firstdetails">
+          <div class="img-container">
+            <img class="image" src="${movie.image.medium}">
+          </div>
+          <div class="show-summary">
+            <h2 class="pop-name">${movie.name}</h2>
+            <p>${movie.summary}</p>
+            <br>
+            <ul class="info">
+              <li>Language: ${movie.language}</li>
+              <li>runtime: ${movie.averageRuntime} minutes</li>
+            </ul>
+            <ul class="info">
+              <li>Rating: ${movie.rating.average}/10</li>
+              <li>premiered: ${movie.premiered}</li>
+            </ul>               
+          </div>
+        </div>
+        <h2  id="new-comment">Comments(<span class="count"></span>)</h2>    
+        <div id="pop-bottom">       
+          <form class="form">
+            <h2 class="add-title">Add a comment</h2> 
+            <input type="text" name="add-name" id="add-name" placeholder="Your name" required> 
+            <textarea type="text" name="add-insight" id="add-comment" placeholder="Your insights" required></textarea>
+            <div class="add-container"> 
+              <button class="add-btn" type="submit">Comment</button>
+            </div>
+          </form>
+        </div>
+      </article>
+    `;
+      projectDetails.innerHTML = popupDetail;
+      const closePopup = projectDetails.querySelector('[data-close-button]');
+      closePopup.addEventListener('click', () => {
+        projectDetails.classList.remove('showpopup');
+        main.classList.remove('blur');
+      });
+
+      const commentList = document.createElement('ul');
+      commentList.classList.add('comment-list');
+      const commentNumber = document.querySelector('.count');
+
+      getMovieComments(button.id).then((response) => {
+        // console.log(response);
```

Comment:
- [ ] Please remove the commented out line to keep the whole code clean and profession